pkgs/games/minecraft/default.nix

```diff
@@ -49,6 +45,10 @@ in stdenv.mkDerivation {
 
     mkdir -p $out/share/applications
     ln -s ${desktopItem}/share/applications/* $out/share/applications/
+
+    ${openjdk}/bin/jar xf $out/minecraft.jar favicon.png
+    mkdir -p $out/share/icons/hicolor/32x32/apps/
+    cp favicon.png $out/share/icons/hicolor/32x32/apps/minecraft.png
   '';
```

Comment:
Thanks for the suggestion. It looks like you have the arguments to install flipped. I will try to update it later.